ui/test/nuxeo-date-picker.test.js

```diff
@@ -97,5 +116,26 @@ suite('nuxeo-date-picker', () => {
         expect(localEltValue.second()).to.be.equal(19);
       });
     });
+
+    suite(!conf.timezone ? 'locale with no timezone' : `locale with ${conf.timezone} timezone`, () => {
+      let currentLocale;
+
+      setup(async () => {
+        currentLocale = moment.locale();
+        element = await makeDatePicker(conf.timezone);
+      });
+
+      teardown(() => {
+        moment.locale(currentLocale);
+      });
+
+      test('the input changes with locale', () => {
+        // using the arabic locale
+        const locale = 'ar';
```

Comment:
co we need this variable?

Reply:
Just to avoid repeating the 'ar' string twice 😄